.github/workflows/lhci.yml

```diff
@@ -0,0 +1,21 @@
+name: Build project and Run Lighthouse CI
+on: [push]
+jobs:
+  lhci:
+    name: Lighthouse CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
checkout has v2 that fixes many issues like [this one](https://github.com/actions/checkout/issues/23), maybe we could use v2?

Reply:
great call! meant to do that.